contrib/local.mk.example

```diff
@@ -9,7 +9,7 @@
 
 # By default, the jemalloc family of memory allocation functions are used.
 # Uncomment the following to instead use libc memory allocation functions:
-# CMAKE_EXTRA_FLAGS="-DUSE_JEMALLOC=OFF"
+# DEPS_CMAKE_FLAGS := -DUSE_BUNDLED_JEMALLOC=OFF
```

Comment:
I think I have a better idea of what's going on now; I wasn't aware of how tricky this could be...

> That depends on how you look at it. :-) It's certainly consistent with how the other dependencies are built and included, and they communicate different things

"Inconsistency" was the wrong word, I suppose I meant minor annoyance. Anyways, thanks to both of you for providing clarification.